floris/tools/uncertainty_interface.py

```diff
@@ -468,46 +225,39 @@ def get_farm_power(self, turbine_weights=None):
                 turbines to 0.0. The array of turbine powers from floris
                 is multiplied with this array in the calculation of the
                 objective function. If None, this  is an array with all values
-                1.0 and with shape equal to (n_wind_directions, n_wind_speeds,
-                n_turbines). Defaults to None.
+                1.0 and with shape equal to (n_findex, n_turbines).
+                Defaults to None.
 
         Returns:
-            NDArrayFloat: Expectation of power production of the wind farm.
-            This array has the shape (num_wind_directions, num_wind_speeds).
+            float: Sum of wind turbine powers in W.
         """
 
         if turbine_weights is None:
             # Default to equal weighing of all turbines when turbine_weights is None
             turbine_weights = np.ones(
                 (
-                    self.floris.flow_field.n_wind_directions,
-                    self.floris.flow_field.n_wind_speeds,
-                    self.floris.farm.n_turbines
+                    self.n_unexpanded,
+                    self.floris.farm.n_turbines,
```

Comment:
Could be on one line.

Reply:
somehow ruff set it this way?

Reply:
It's the trailing comma. If you have a comma at the end of a sequence, ruff is configured to split that. If you delete the comma after "n_turbines", you can make it one line and satisfy ruff.